src/Tribe/Importer/File_Importer_Events.php

```diff
@@ -482,4 +500,20 @@ private function get_currency_position( array $record ) {
 
 		return 'prefix';
 	}
+
+	/**
+	 * Returns the `post_excerpt` to use.
+	 *
+	 * @deprecated TBD
+	 *
+	 * @param int    $event_id        The event id being updated by import.
+	 * @param string $import_excerpt The imported excerpt text.
+	 *
+	 * @return string
+	 */
+	private function get_post_excerpt( $event_id, $import_excerpt ) {
+		_deprecated_function( __METHOD__, 'TBD', '$this->get_post_text_field( $event_id, $record, 'event_excerpt', 'post_excerpt' )' );
```

Comment:
* :no_entry_sign: PHP syntax error: syntax error, unexpected 'event_excerpt' (T_STRING), expecting ',' or ')'
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'event_excerpt'
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'event_excerpt'.

<sub>hash: 74857753ded22b928d6bbbd93ce50968</sub>